prototypes/basic/app/views/upload.html

```diff
@@ -4,28 +4,64 @@
 
 {% block pageTitle %} {{ serviceName }} – GOV.UK Prototype Kit {% endblock %}
 {% block beforeContent %}
-{{ govukBackLink({ text: "Back", href: "javascript:window.history.back()" }) }}
+{{
+    govukBreadcrumbs({
+      items: [
+        {
+          text: "Home",
+          href: "/"
+        },
+        {
+            text: "Upload spreadsheet",
+            href: "/upload"
+        }
+      ]
+    })
+  }}
 {% endblock %}
 
 {% block content %}
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-l">Upload your data</h1>
 
+        <p class="govuk-body">
+          The expected pay period is from <strong>{{ "start" | lastMonthRange }}</strong> to <strong>{{ "end" | lastMonthRange }}</strong>,
+          and all of the dates in this file should be in that range (inclusive).
+          Any rows with dates outside of that range will not be processed.
+        </p>
+
+        <div class="govuk-inset-text">
+          <h1 class="govuk-heading-s">Check your dates</h1>
+          <p class="govuk-body">
+            Before you upload the data, you should check the dates are formatted as yyyy-mm-dd (e.g. 2025-02-28).
+          </p>
+        </div>
+
+        <p class="govuk-body">
+          The file you upload must be in one of the following formats, but you will be given the opportunity to choose which
+          sheet contains the data, and which columns contain the required fields.
+          <ul class="govuk-list govuk-list--bullet">
+            <li>Comma-separated values - CSV</li>
+            <li>Microsoft Office - XLSX or XLS</li>
+            <li>Open office document - ODS</li>
```

Comment:
This might be better referred to as "OpenDocument Spreadsheet"?

Reply:
Done in 848554f